java/kafka/deployment-oauth.yaml

```diff
@@ -219,7 +219,7 @@ spec:
                   name: sso-x509-https-secret
                   key: tls.crt
             - name: KAFKA_BOOTSTRAP_SERVERS
-              value: my-cluster-kafka-bootstrap:9093
+              value: my-cluster-kafka-bootstrap:9092
```

Comment:
Why did you change this? Given the `KAFKA_SECURITY_PROTOCOL` and `KAFKA_SSL_TRUSTSTORE_*` values I would say that 9093 fits better with our operator examples.